build.gradle

```diff
@@ -137,6 +137,7 @@ configure(javaCodeCheckedProjects) {
                 'org.mongodb.async.type': System.getProperty('org.mongodb.async.type', 'nio2'),
                 'org.mongodb.test.awsAccessKeyId': System.getProperty('org.mongodb.test.awsAccessKeyId'),
                 'org.mongodb.test.awsSecretAccessKey': System.getProperty('org.mongodb.test.awsSecretAccessKey'),
+                'org.mongodb.test.mongocryptdSpawnPath': System.getProperty('org.mongodb.test.mongocryptdSpawnPath'),
```

Comment:
Without this, the system property never makes it into the test

Reply:
We should simplify this and add all system properties that start with `org.mongodb`

Reply:
OK to leave it like this for this PR?